core/kazoo_apps/src/kapps_speech.hrl

```diff
@@ -49,7 +49,7 @@
         ,{<<"male/tr-tr">>, <<"eurturkishmale">>}
         ]
        ).
--define(ISPEECH_TTS_URL, kapps_config:get_string(?MOD_CONFIG_CAT, <<"tts_url">>, <<"http://api.ispeech.org/api/json">>)).
+-define(ISPEECH_TTS_URL, kapps_config:get_string(?MOD_CONFIG_CAT, <<"tts_url_ispeech">>, <<"http://api.ispeech.org/api/json">>)).
```

Comment:
Are you including a migrate command for this? It is possible that there are other values (I doubt it, but would rather address is than get yelled at that we broke someone's TTS engine).